app/components/support_interface/tile_component.rb

```diff
@@ -0,0 +1,29 @@
+module SupportInterface
+  class TileComponent < ActionView::Component::Base
+    attr_reader :count, :label
+
+    def initialize(count:, label:, tile_type: :regular, size: :regular)
+      @count = count
+      @label = label
+      @tile_type = tile_type
+      @size = size
+    end
+
+    def regular_type?
+      @tile_type == :regular
+    end
+
+    def type_colour
+      case @tile_type
+      when :headline then 'blue'
+      when :success then 'green'
+      when :error then 'red'
+      else ''
+      end
+    end
```

Comment:
`TagComponent` was changed to not use a semantic mapping and instead just ask the user to pass in the desired colour: https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1018

I'm impartial to either approach and think either can work 👍